backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/price/internal/BpkPriceImpl.kt

```diff
@@ -46,7 +49,11 @@ internal fun BpkPriceAlignStart(
                 BpkText(
                     text = it,
                     color = BpkTheme.colors.textSecondary,
-                    style = if (size == BpkPriceSize.Large) BpkTheme.typography.footnote else BpkTheme.typography.caption,
+                    style = when (size) {
+                        BpkPriceSize.Large -> BpkTheme.typography.footnote
+                        BpkPriceSize.Small -> BpkTheme.typography.caption
+                        BpkPriceSize.ExtraSmall -> BpkTheme.typography.caption
```

Comment:
nit: is this change necessary?

Reply:
I think it's good to use a when statement here, however maybe you can simplify it by having the small & extrasmall in the same case. Also as the same style is used across different variations of this could you extract it to a separate (private) function/property & reuse across all of them?